contrib/ansible/group_vars/all.yml

```diff
@@ -6,7 +6,7 @@ cluster_name: cluster.local
 # Account name of remote user. Ansible will use this user account to ssh into
 # the managed machines. The user must be able to use sudo without asking
 # for password unless ansible_sudo_pass is set
-#ansible_ssh_user: root
+ansible_ssh_user: fedora
```

Comment:
All.yml is the override. Roles can have vars and defaults. Look up The Ansible ordering for variables. Command line is what vagrant is using in extra vars. so any var or default in a role can be overridden in group_vars